packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -74,20 +81,20 @@ export const PaymentConfirmation = ({
         <div className="bottom-row">
           <Localized
             id="payment-confirmation-amount"
-            amount={formatCurrencyInCents(amount)}
-            interval={interval}
+            $amount={formatCurrencyInCents(amount)}
+            $interval={interval}
           >
             <p></p>
           </Localized>
-          <Localized id="payment-confirmation-cc-preview" last4={last4}>
+          <Localized id="payment-confirmation-cc-preview" $last4={last4}>
             <p className={`c-card ${brand.toLowerCase()}`}></p>
           </Localized>
         </div>
       </div>
 
       <div className="footer">
         <Localized id="payment-confirmation-download-button">
-          <a>click to download</a>
+          <a href={redirectUrl}>click to download</a>
```

Comment:
Are we going to be automatically redirecting users to this URL? If not, maybe we call it `downloadUrl`?